app/components/search/current_filters_component.html.erb

```diff
@@ -0,0 +1,8 @@
+<section aria-label="Current Filters">
+  <h2>Current Filters</h2>
+  <ul>
+    <% current_filters.each do |form_field, value| %>
+      <li><strong><%= form_field.humanize %>:</strong> <%= value %> <%= link_to 'Remove', search_items_path(search_form.without_attributes({ form_field => value, page: nil })) %></li>
+    <% end %>
```

Comment:
We could also consider extracting a tiny `CurrentFilterComponent` for this logic and then replacing this code with something like:

```ruby
    <%= render CurrentFilterComponent.with_collection(current_filters, search_form:) %>
```

But we don't need to do that here or now.